modules/quant_search/quant_search.routing.yml

```diff
@@ -5,8 +5,6 @@ quant_search.main:
     _title: 'Quant Search'
   requirements:
     _permission: 'administer quant search'
-  options:
-    _admin_route: TRUE
```

Comment:
I'm torn, I like declarative and explicit definitions - but default behaviour is to set this to `TRUE` for routes starting with `/admin` - unlikely that this will change from core though, so happy to remove.

Reply:
Yeah... I only removed because others didn't have them so I checked and the default is TRUE. If we keep then we should be consistent and use for them all.